modules/settings/Settings.js

```diff
@@ -1,191 +0,0 @@
-/* global JitsiMeetJS */
-const logger = require('jitsi-meet-logger').getLogger(__filename);
-
-import UIUtil from '../UI/util/UIUtil';
-import jitsiLocalStorage from '../util/JitsiLocalStorage';
-import { randomHexString } from '../../react/features/base/util';
-
-let avatarUrl = '';
-
-let email = UIUtil.unescapeHtml(jitsiLocalStorage.getItem('email') || '');
```

Comment:
Can UIUtil.unescapeHtml be killed off now?

Reply:
"escape" and "unescape" exist and lodash is a dependency and it has tools too. UIUtil.unescapeHtm

Reply:
Let's keep it on web, it it's only used there.

Reply:
It's not used anywhere on web. It's defined and not used.

Reply:
I'm happier with a cross platform solution so I'd go for lodash

Reply:
Sure, if we need it, let's use lodash then!